build_tools/waf_dynamo.py

```diff
@@ -1195,33 +1243,36 @@ def do_find_file(file_name, path_list):
 def find_file(self, file_name, path_list = [], var = None, mandatory = False):
     if not path_list: path_list = os.environ['PATH'].split(os.pathsep)
     ret = do_find_file(file_name, path_list)
-    self.check_message('file', file_name, ret, ret)
+
+    # JG: Maybe fix this. Not sure it's the correct conversion to 'check_message'
+    self.start_msg('Checking for file %s' % (file_name))
+    self.end_msg(ret)
```

Comment:
'check_message' is removed in waf now, I think this is correct but might need an extra verification. Not the end of the world however.